libs/ui-lib/lib/ocm/components/clusterWizard/ClusterWizardFooter.tsx

```diff
@@ -55,6 +55,7 @@ const ValidationSection = ({
           hosts={cluster?.hosts || []}
           validationsInfo={validationsInfo}
           wizardStepsValidationsMap={wizardStepsValidationsMap}
+          lastInstallationPreparation={cluster['last-installation-preparation']}
```

Comment:
Can this error also happen on CIM? If that's the case, it would be important to also show it there.

Reply:
CIM uses completely different data and has a different way of handling the errors.

Reply:
CIM uses a different API. This errors only appears in OCM side